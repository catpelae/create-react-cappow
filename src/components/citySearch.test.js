import React from 'react';
import renderer from 'react-test-renderer';
import CitySearch from './citySearch';

it('renders correctly', () => {
 const tree = renderer.create(
   <CitySearch />
 ).toJSON();
 expect(tree).toMatchSnapshot();
});