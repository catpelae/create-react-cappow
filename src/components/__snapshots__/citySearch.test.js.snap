// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<form
  className="CitySearch-main"
  onSubmit={[Function]}
>
  <label
    className="CitySearch-search"
  >
     Type in cities to compare separated by commas
  </label>
  <input
    className="CitySearch-input form-control"
    onChange={[Function]}
    placeholder="Paris, Bogota"
    type="text"
    value=""
  />
  <input
    className="CitySearch-button"
    type="submit"
    value="Submit"
  />
</form>
`;
